libs/agno/agno/workflow/workflow.py

```diff
@@ -2782,6 +2789,12 @@ def _prepare_steps(self):
                     step_name = step.name or f"step_{i + 1}"
                     log_debug(f"Step {i + 1}: Team '{step_name}' with {len(step.members)} members")
                     prepared_steps.append(Step(name=step_name, description=step.description, team=step))
+                if isinstance(step, Step) and step.add_workflow_history is True and self.db is None:
+                    log_warning(
+                        f"Step '{step.name or f'step_{i+1}'}' has add_workflow_history=True but "
+                        "the Workflow has no database configured. Workflow history will not persist. "
+                        "Add db=SqliteDb() or another database to the Workflow."
+                    )
```

Comment:
```suggestion
                    log_warning(
                        f"Step '{step.name or f'step_{i+1}'}' has add_workflow_history=True "
                        "but no database is configured in the Workflow. "
                        "History won't be persisted. Add a database to persist runs across executions."
                    )
```

Just keeping the logs aligned